src/cmd/create.rs

```diff
@@ -109,18 +119,17 @@ impl Basic {
 
 impl Sharded {
     pub async fn run(&self, opts: Opts) -> Result {
-        let seed_words = if self.seed {
-            Some(get_seed_words()?)
-        } else {
-            None
+        let seed_words = match self.seed {
+            Some(seed_type) => Some(get_seed_words(seed_type == SeedType::Bip39)?),
```

Comment:
Is there a better way to do this `== Enum::Value` in rust? I didn't wan `get_seed_words()` or `gen_keypair()` to have to know about the Enum type, I just wanted to pass a bool. But, this made me add several traits to the enum definition just to do this comparison. 

Please let me know if there is something better to do here.

Reply:
I would pass the enum type through all of these functions instead of maintaining this bool. So `fn get_seed_words(seed_type: SeedType)`. Do the same for `mnemonic_to_entropy`. You'll see your `if` statements all become `matches` and it'll be much easier to read IMO.

Reply:
Done and pushed. I like this much better!

Reply:
![image](https://user-images.githubusercontent.com/1606640/119526835-9921a480-bd34-11eb-91c7-9e32937c9e22.png)